src/style/themes/base/base-theme.config.js

```diff
@@ -20,7 +20,7 @@ export default (palette) => {
 
       // status
       error: palette.errorRed,
-      focus: palette.gold,
+      focus: palette.goldTint(50),
```

Comment:
This should not be changed. What is this `focus` going to be used for?

Reply:
right, I added this to the small, medium and large theme instead

Reply:
I would probably give this a more specific name, like `linkFocus`. Otherwise it could be describing a focus state for anything.